### DISCUSSION TITLE
limit query parameter for dagruns is not working properly when given value above 100

### DISCUSSION THREAD

**Discussion by msoni1369** [apache/airflow#32025] [Q&A] [answered]
The limit query parameter in the get_dags API is not working properly. The default value is 100 when a value is given lower than 100 it works but when a value is given higher than 100 it shows only 100. By default, the value is 100, the same is mentioned in the documentation as well.  
But it is not mentioned that it is the max value. How can I get all dags with get_dags API.  Any suggestions would be helpful.

https://airflow.apache.org/docs/apache-airflow/stable/stable-rest-api-ref.html#operation/get_dags

**Comment by ephraimbuddy** [comment, marked as answer]
You need to update the maximum page limit using this configuration: `AIRFLOW__API__MAXIMUM_PAGE_LIMIT`
See: https://airflow.apache.org/docs/apache-airflow/stable/configurations-ref.html#maximum-page-limit

**Reply by msoni1369** [reply]
Thanks for your response.
